e2e/src/usecases/DappListRecent.js

```diff
@@ -0,0 +1,180 @@
+import { reloadReactNative } from '../utils/retries'
+import { sleep, waitForElementId, getElementText, getElementTextList } from '../utils/utils'
+
+const jestExpect = require('expect')
+
+/**
+ * From the home screen, navigate to the dapp explorer screen
+ * Disabled detox device sync due to dapp list double fetch / reanimated issue
+ * https://github.com/wix/Detox/issues/2799
+ */
+async function navigateToDappList() {
+  await device.disableSynchronization()
+  await waitForElementId('Hamburger')
+  await element(by.id('Hamburger')).tap()
+  await waitForElementId('dapps-explorer-icon')
+  await element(by.id('dapps-explorer-icon')).tap()
+  await waitFor(element(by.id('DAppExplorerScreen/loading')))
+    .not.toBeVisible()
+    .withTimeout(30 * 1000)
+  await sleep(5 * 1000)
```

Comment:
could we replace this waitFor loader and sleep with something like "wait for there to be elements with test id `DappCard`"? since the loader appears asynchronously (triggered by a saga) there might be a race condition where sometimes line 17 evaluates to true already before the loader has had a chance to appear...it'd also be nice not to wait 5 seconds if we can avoid it :)

Reply:
Aadjusted and removed the sleep.
```
  await waitFor(element(by.id('DAppExplorerScreen/DappsList')))
    .toExist()
    .withTimeout(10 * 1000)
```